workspaces/build.rs

```diff
@@ -1,8 +1,8 @@
 fn main() {
     let doc_build = cfg!(doc) || std::env::var("DOCS_RS").is_ok();
-    if !doc_build && cfg!(feature = "install") {
+    let env_bin = std::env::var("NEAR_SANDBOX_BIN_PATH").is_ok();
```

Comment:
why is this additional check needed?

Reply:
just thought it'd be nice to have if a user specified the env var so as to not trigger an install if not necessary, but I probably should add the if-env-var changed, a rebuild should occur. Would you see any potential footguns with this?

Reply:
I thought this was already checked? I guess it doesn't hurt to have redundancy to make sure it's short-circuited